casper/src/main/scala/coop/rchain/casper/util/BondingUtil.scala

```diff
@@ -27,7 +27,7 @@ object BondingUtil {
 
   def bondingForwarderDeploy(bondKey: String, ethAddress: String): String =
     s"""new rl(`rho:registry:lookup`), SystemInstancesCh, posCh in {
-       |  rl!(`rho:id:wdwc36f4ixa6xacck3ddepmgueum7zueuczgthcqp6771kdu8jogm8`, *SystemInstancesCh) |
+       |  rl!(`rho:id:systemInstancesRegistry`, *SystemInstancesCh) |
```

Comment:
Not related to this work, but I thought this contract was supposed to be removed.

Reply:
@odeac